neuland/digitizing/R3BDigitizingChannel.h

```diff
@@ -68,25 +79,42 @@ namespace R3B::Digitizing
 
         virtual void AddHit(Hit hit) = 0;
         virtual auto HasFired() -> bool;
+        // virtual auto HasCalFired() -> bool;
 
         // Getters:
         virtual auto GetTrigTime() -> double;
         auto GetSignals() -> const Signals&;
         auto GetSide() const -> ChannelSide { return fSide; }
         auto GetPaddle() const -> Paddle* { return fPaddle; }
+        virtual auto GetCalSignals() -> CalSignals
+        {
+            auto virtual_sig = CalSignals{};
+            return virtual_sig;
+        }
```

Comment:
```suggestion
        virtual auto GetCalSignals() -> CalSignals
        {
            return {};
        }
```

Normally, a getter function returns a value/const ref to one of its member variables. Which member variable of this base class are you trying to return?